### ISSUE TITLE
GUI - When adding too many executables "Gear Icon" to go to main list with all executables is no longer visible

### ISSUE THREAD

**Issue by mikelpapamike** [rrebuffo/CasparLauncher#15]
Thanks a lot for that great tool for us the CasparCG users. I tried adding too many executables and at some point the gear icon is no longer accessible due to many added tabs next to it,

**Comment by rrebuffo** [comment]
Even though it should be always docked to the right and that's clearly a bug, you can always use the keyboard to switch between the tabs. 
The tab strip also supports the mouse scroll wheel.

**Comment by mikelpapamike** [comment]
Thanks for letting me know , I made tests of it on a laptop , and the laptop scroll gesture does not seem to scroll the tabs .

**Comment by rrebuffo** [comment]
Try the vertical gesture or pressing shift while you do it

**Comment by mikelpapamike** [comment]
Yes the vertical scroll works with or without shift. And TAB always goes to the Main tab which is good. Thank you so much
